e107_admin/plugin.php

```diff
@@ -665,51 +680,59 @@ private function pluginDownload()
 		$frm = e107::getForm();
 		$mes = e107::getMessage();
 		
+		e107::getDebug()->log("Plugin Download.");
+		$tmp = explode('&amp;', e_QUERY); // #1712 we need to process new ampersand separated query
```

Comment:
Best way to learn is have a go. 
What I found was $_GET only contained mode=download, there was no other data in there.
src is encrypted and in e_QUERY and that $tmp was already trying to grab values from exploding e_QUERY using .dot. separator. 
However, we got &amp; separators. So I added check to see id e_query contained &'s then exploded with that.